### ISSUE TITLE
Clarification regarding the training dataset

### ISSUE THREAD

**Issue by priyarana25** [hliulab/UnifyImmun#6]
Hello sir, in the training data for true dataset with label 1 the sequence used was the region binding (showing hits) but for the false dataset with label 0 what was the basis for the sequence used and how was it generated? And is this the negative data sample generated as stated in the paper?
